datalad_next/create_sibling_webdav.py

```diff
@@ -0,0 +1,547 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 et:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""High-level interface for creating a combi-target on a WebDAV capable server
+ """
+import logging
+import os
+from typing import (
+    Optional,
+    Union,
+)
+from urllib.parse import urlparse
+
+from datalad.distribution.dataset import (
+    Dataset,
+    EnsureDataset,
+    datasetmethod,
+    require_dataset,
+)
+from datalad.downloaders.credentials import UserPassword
+from datalad.interface.base import Interface
+from datalad.interface.base import (
+    Interface,
+    build_doc,
+)
+from datalad.interface.common_opts import (
+    recursion_flag,
+    recursion_limit
+)
+from datalad.interface.results import get_status_dict
+from datalad.interface.utils import eval_results
+from datalad.support.param import Parameter
```

Comment:
`log_progress()` is used below, but never imported.

```suggestion
from datalad.log import log_progress
from datalad.support.param import Parameter
```